api/workAllocation2/util.ts

```diff
@@ -482,7 +488,7 @@ export function getSubstantiveRoles(roleAssignments: RoleAssignment[]): RoleAssi
 }
 
 // Note: array type may need to be changed depending on where pagination called
-export const paginate = (array: Case[], pageNumber: number, pageSize: number): any[] => {
+export const paginate = (array: RoleCaseData[], pageNumber: number, pageSize: number): any[] => {
```

Comment:
Paginate is a generic function. Can we can this take an `array` generic type. Example code below

`export function paginate<T>(array: T[], pageNumber: number, pageSize: number): T[] {
  return array.slice((pageNumber - 1) * pageSize, pageNumber * pageSize);
};`

Reply:
good point, will do